pkg/importer/dataStream.go

```diff
@@ -854,6 +857,10 @@ func (d *DataStream) isHTTPQcow2() bool {
 
 func (d *DataStream) calculateTargetSize(dest string) int64 {
 	targetQuantity := resource.NewScaledQuantity(d.AvailableDestSpace, 0)
+	if d.VolumeMode == v1.PersistentVolumeBlock {
```

Comment:
This shouldn't be needed, in the importer.go when we populate the dso.AvailableDestSpace we call util.GetAvailableSpace, in that function we check if we have a block device, if so we return util.GetAvailableSpaceBlock, which is the same as you are doing here. So d.AvailableDestSpace should already have the correct value.

Reply:
Can you point me where we check this is block device? Can’t fint it in Util.GetAvailableSpace.

Reply:
Oh... it’s in the importer. Ok

Reply:
No. This is wrong. Still need you to point me where we check for block device.
I only see in the importer 
AvailableDestSpace: util.GetAvailableSpace(common.ImporterVolumePath)
This one does not check for block device.
Did you meant to change this to call the GetAvailableSpace defined as last method in the importer?

Reply:
No in the importer we call GetAvailableSpace, which will return the available space of the destionation (either a fs or a block). That value is then stored in d.AvailableDestSpace, which we can use over here.